huaweicloud/services/dli/resource_huaweicloud_dli_queue.go

```diff
@@ -275,6 +318,17 @@ func resourceDliQueueRead(_ context.Context, d *schema.ResourceData, meta interf
 		d.Set("elastic_resource_pool_name", queueDetail.ElasticResourcePoolName),
 	)
 
+	if elasticResourcePoolName, ok := d.GetOk("elastic_resource_pool_name"); ok {
+		policies, err := getQueueScalingPolicies(cfg, region, elasticResourcePoolName.(string), queueName)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+
+		mErr = multierror.Append(
+			d.Set("scaling_policies", policies),
+		)
```

Comment:
```suggestion
		mErr = multierror.Append(mErr, d.Set("scaling_policies", policies))
```
please append setting error to the exist error array

Reply:
fixed